dotcom-rendering/src/lib/useIsMyGuardianEnabled.ts

```diff
@@ -0,0 +1,18 @@
+import { log } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+import { getEnvironmentClient } from './bridgetApi';
+
+export const useIsMyGuardianEnabled = (): boolean | undefined => {
+	const [isEnabled, setIsEnabled] = useState<boolean | undefined>(undefined);
```

Comment:
Why do we need to have the three states `true`, `false` and `undefined`? Can we simplify this to be a true boolean?

```ts
	const [isEnabled, setIsEnabled] = useState(false);
```

From what I can see we collapse the falsy cases already.